source/containers.tex

```diff
@@ -6338,7 +6338,7 @@
 \pnum
 \requires
 \tcode{is_assignable_v<mapped_type\&, M\&\&>} shall be \tcode{true}.
-\tcode{value_type} shall be \tcode{EmplaceConstructible} into \tcode{map}
+\tcode{value_type} shall be \tcode{Emplace\-Constructible} into \tcode{map}
```

Comment:
Ah, I just tried, and we probably really don't want to monkey with global `tcode`, because then we'd end up hyphenating things like `1.2e-34`. :-)